bin/improver-threshold

```diff
@@ -45,17 +46,28 @@ def main():
         description="Calculate the threshold truth value of cube data "
         "relative to the provided threshold value. By default data are "
         "tested to be above the thresholds, though the --below_threshold "
-        "flag enables testing below thresholds. A fuzzy factor may be "
-        "provided to capture data that is within this factor of the "
+        "flag enables testing below thresholds. A fuzzy factor or fuzzy "
+        "bounds may be provided to capture data that is close to the "
         "threshold.")
     parser.add_argument("input_filepath", metavar="INPUT_FILE",
                         help="A path to an input NetCDF file to be processed")
     parser.add_argument("output_filepath", metavar="OUTPUT_FILE",
                         help="The output path for the processed NetCDF")
     parser.add_argument("threshold_values", metavar="THRESHOLD_VALUES",
-                        nargs="+", type=float,
+                        nargs="*", type=float,
                         help="Threshold value or values about which to "
-                        "calculate the truth values; e.g. 270 300")
+                        "calculate the truth values; e.g. 270 300. "
+                        "Must be omitted if --threshold_config is used.")
+    parser.add_argument("--threshold_config", metavar="THRESHOLD_CONFIG",
```

Comment:
Doesn't currently object to duplicate keys (e.g. a repeated threshold with different bounds). The JSON module just takes the last instance in the input file. I'm not convinced this will ever raise its head, and we would have to use a custom load to account for it. I'm not asking for this, but you should feel free to implement it if for robustness if you would like.

Reply:
I've added a note to the CLI help. I'll spend a few minutes seeing if anything can be done about this, but it will be a separate PR.